boards/Makefile.common

```diff
@@ -168,6 +169,7 @@ endif
 SIZE      ?= $(TOOLCHAIN)-size
 OBJCOPY   ?= $(TOOLCHAIN)-objcopy
 OBJDUMP   ?= $(TOOLCHAIN)-objdump
+RUSTC_SYSROOT ?= "$(shell rustc --print sysroot)"
```

Comment:
Does this need to be configurable? Can we move it earlier in the file (it's not a tool)?

Reply:
Does not need to be configurable, I fixed that and moved it earlier.